diagnostic.md

```diff
@@ -3,36 +3,54 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+- Sets up a front end saving the developer from a lot of coding
+- has structured folder layout and file naming conventions making it better for
+  leaning.
+- aids in making front end reliable from earlier on in development process
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
 ```sh
-# your answer here
+ember uses handlebars as a templating system. So its seems to be more than just
+related according to ember docs.
+
+"Ember.js uses the Handlebars templating library to power your app's user
+interface."
+-ember docs
+https://guides.emberjs.com/v1.10.0/templates/handlebars-basics/
```

Comment:
Yes, Ember uses handlebars, but it also enhances handlebars with some built-in helpers like `link-to`, and the ability to use components in templates.
.5/1